packages/selleo-design-core/src/Tabs.tsx

```diff
@@ -0,0 +1,25 @@
+import { h } from "preact";
+import cx from "classnames";
+
+type TabProps = {
+  active: boolean;
+  icon?: JSX.Element;
+};
+
+export function Tab({ active, icon }: TabProps) {
+  const classes = cx(
+    "px-2 py-1 text-base font-normal rounded-lg cursor-pointer",
+    {
+      "text-[#FF6D2A] bg-[#FFF8F5]": active,
+      "text-[#4C4D52] bg-white hover:bg-[#FFF8F5] dark:text-white dark:bg-[#2F3033] dark:hover:bg-[#4C4D52]":
```

Comment:
please use the colors that are initated in tailwind.config.cjs. They're from the design system core and should be the only one that we use

Reply:
Yes, you'are right